### DISCUSSION TITLE
Parameter "type" in setError is not explained anywhere. What does it mean?

### DISCUSSION THREAD

**Discussion by NullPainter2** [react-hook-form/react-hook-form#3042] [Q&A]
Hi,

I am having trouble finding what parameter "type" in setError() means. I was not able to find it anywhere in docs (V5 or V6). What does it mean?

If it is just for distinction between multiple errors, please add this to documentation.

Thanks :)

**Comment by codenoobforreal** [comment]
I think it is used to distinguish between different types of errors in the same field
e.g. errors.username.errortype1 errors.username.errortype2
